scripts/eval_bop24_pose.py

```diff
@@ -55,7 +55,7 @@
     "eval_path": config.eval_path,
     # File with a list of estimation targets to consider. The file is assumed to
     # be stored in the dataset folder.
-    "targets_filename": "test_targets_bop19.json",  # TODO: change to "test_targets_bop24.json"
+    "targets_filename": "test_targets_bop24.json",
```

Comment:
@thodan @nv-nguyen do we actually want to change this name? It would need an update of all datasets.

Reply:
nevermind, it is required because we remove the `"inst_count": 1, "obj_id": 4`